Puzzle/src/org/fundacionjala/devops/bootcamp/Move/Movement.java

```diff
@@ -0,0 +1,182 @@
+package org.fundacionjala.devops.bootcamp.Move;
+
+public class Movement {
+
+    private Position position;
+    private String direction;
+    int size;
+
+    public Movement(Position position, String direction) {
+        this.position = position;
+        this.direction = direction;
+    }
+
+    public Movement() {
+
+    }
+
+    public Position getPosition() {
+        return position;
+    }
+
+    public String getDirection() {
+        return direction;
+    }
+
+    public void setPosition(Position position) {
+        this.position = position;
+    }
+
+    public void setDirection(String direction) {
+        this.direction = direction;
+    }
+
+
+    //Method to check if the movement is valid:
+    public boolean isValidMovement(String position, String direction,int size) {
```

Comment:
As we are validating some movements, maybe we can create an abstract class called "MovementValidator"